package.json

```diff
@@ -55,7 +56,8 @@
     "hoist-non-react-statics": "^1.2.0",
     "immutable": "^3.8.1",
     "jwt-decode": "^2.2.0",
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.4",
+    "opencollective": "^1.0.3"
```

Comment:
Can we make this a dev dependency instead? Some users like dependencies to be only things used specifically in production.